pages/form/TravelMethod.jsx

```diff
@@ -78,8 +78,21 @@ export default function Question4() {
                 Select the ways you generally travel to work.
               </Text>
             </Box>
-            <TravelMethodButtons handleTransportMode={handleTransportMode} />
-
+            <TravelMethodButtons handleTransportMode={handleTransportMode}  />
+{/* Collapse on carpool selection*/}
+<Collapse in={transportMode === "Carpool"}>
+          <Text
+            mb={5}
+            fontSize={[15, 17]}
+            px="20px"
+            py="12px"
+            color="#155724"
+            bg="#D4EDDA"
+          >
+           How many other people would you most likely to carpool with?
+          </Text>
+        </Collapse>
```

Comment:
Have you forgotten to add the `Counter` component here?
```suggestion
        </Collapse>
        <Counter handleMinus handlePlus count />
```